src/dsp_tools/commands/excel2json/properties.py

```diff
@@ -269,49 +281,34 @@ def _get_gui_attribute(
 ) -> GuiAttributes | InvalidExcelContentProblem | None:
     if pd.isnull(df_row["gui_attributes"]):
         return None
-    # If the attribute is not in the correct format, a called function may raise an InputError
     try:
-        return _format_gui_attribute(attribute_str=df_row["gui_attributes"])
-    except InputError:
+        return _unpack_gui_attributes(attribute_str=df_row["gui_attributes"])
+    except InvalidGuiAttributeError:
         return InvalidExcelContentProblem(
-            expected_content="attribute1: value, attribute2: value (no attribute key may be duplicated)",
+            expected_content="The only valid gui-attribute is 'hlist' for the gui-element 'List'.",
             actual_content=df_row["gui_attributes"],
             excel_position=PositionInExcel(column="gui_attributes", row=row_num),
         )
 
 
-def _format_gui_attribute(attribute_str: str) -> GuiAttributes:
-    attribute_dict = _unpack_gui_attributes(attribute_str=attribute_str)
-    return GuiAttributes(
-        {attrib: _search_convert_numbers_in_str(value_str=val) for attrib, val in attribute_dict.items()}
-    )
-
-
-def _unpack_gui_attributes(attribute_str: str) -> dict[str, str]:
+def _unpack_gui_attributes(attribute_str: str) -> GuiAttributes:
+    err_msg = "The only valid gui-attribute is 'hlist' for the gui-element 'List'."
     gui_list = [x.strip() for x in attribute_str.split(",") if x.strip() != ""]
-    gui_attrib = {}
-    for attrib in gui_list:
-        attrib_key, attrib_val = _extract_information_from_single_gui_attribute(attrib)
-        if attrib_key in gui_attrib:
-            raise InputError("Duplicate gui attribute")
-        gui_attrib[attrib_key] = attrib_val
-    return gui_attrib
+    if not gui_list:
+        return GuiAttributes({})
+    if len(gui_list) > 1:
+        raise InvalidGuiAttributeError(err_msg)
+    attrib_key, attrib_val = _extract_information_from_single_gui_attribute(gui_list.pop(0))
+    if attrib_key != "hlist":
+        raise InvalidGuiAttributeError(err_msg)
+    return GuiAttributes({attrib_key: attrib_val})
```

Comment:
In the class `GuiAttributes`, you can now change the type from `dict[str, int | str | float]`  to `dict[str, str]`